topics/variant-analysis/tutorials/dunovo/tutorial.md

```diff
@@ -345,11 +328,11 @@ Now we'll want to parse the VCF produced by the NVC, determine what the major an
 
 Now we have a file containing the base counts for every site covered by at least 10 reads. We'd like to filter through this data to find sites with a reasonable chance of being a real variant, not sequencing error.
 
-The {% icon tool %} **Variant Annotator** produces a simple tab-delimited file, with one site per line. We can use the {% icon tool %} **Filter** tool from the **Filter and Sort** section to process this kind of file. We'll use the filter `c16 >= 0.01` to remove lines where the value in column 16 is less than 0.01. Column 16 contains the minor allele frequency, so this will remove all sites with a MAF less than 1%.
+The {% icon tool %} **Variant Annotator** produces a simple tab-delimited file, with one site per line. We can use the {% icon tool %} **Filter** tool to process this kind of file. We'll use the filter `c16 >= 0.01` to remove lines where the value in column 16 is less than 0.01. Column 16 contains the minor allele frequency, so this will remove all sites with a MAF less than 1%.
 
 > ### {% icon hands_on %} Hands-on: Filter the raw variants list
 >
-> Run {% icon tool %} **Filter** with the following parameters:
+> Run {% tool [Filter](https://usegalaxy.org/root?tool_id=Filter1) %} with the following parameters:
```

Comment:
this one still has the server url

Reply:
@shiltemann I'm not sure where to find the toolshed ID. When I check the "Job Information" of the item in my history, I see: `Galaxy Tool ID:  Filter1` instead of a toolshed URL. If I do "Get Link" from the tool form, I get: `https://usegalaxy.eu/root?tool_id=Filter1` or `https://usegalaxy.org/root?tool_id=Filter1`. Is there somewhere else I should be looking to find the toolshed URL?

Reply:
@malloryfreeberg This is not a ToolShed tool, it comes with Galaxy: https://github.com/galaxyproject/galaxy/blob/dev/tools/stats/filtering.xml

Reply:
indeed, and it should work if you just add `Filter1` as the tool id

Reply:
Thanks @nsoranzo and @shiltemann. Updated!